docker-compose.yml

```diff
@@ -137,6 +137,30 @@ services:
       - "logging=true"
     restart: always
     logging: *default-logging
+  login-dashboard:
+    image: lblod/acmidm-login-service:0.11.0
```

Comment:
There is a 0.12.0 release now as well, so I would use that (same for the others).